app/assets/javascripts/views/story_view.js

```diff
@@ -212,6 +212,16 @@ module.exports = FormView.extend({
     });
   },
 
+  highlightSearchedStories: function () {
+    let storyID = `#story-${ this.model.get('id') }`;
+    let storyElement = $(storyID);
+    storyElement.effect("highlight", {}, 3500);
+    storyElement.parent().animate({
```

Comment:
Is this indentation okay?

Reply:
Got a little wonky, but now it's all good.